content/raw/chat/activity_center.md

```diff
@@ -0,0 +1,123 @@
+* Clicking "Show notifications that have been read" MUST show all activity items in the selected category
+* Clicking "Hide notifications that have been read" MUST hide activity items in the selected category that have been previously marked as read
+* The activity items MUST be grouped by day sections and sorted by most recent
+* The day sections and their items MUST displayed according to the following criteria
+    * The section with items from today MUST be displayed as "Today"
+    * The section with items from yesterday MUST be displayed as "Yesterday"
+    * The section with items from this year MUST be displayed in the format "Weekday, Day Month" section example: "Mon, 21 April"
+    * The section with items from previous years MUST be displayed in the format "Day Month Year" section example: "Mon, 21 April 2020"
+* The Activity Center MUST display a badge with the total number of currently unread activity items
+* If activity items are are marked as read, the activity center badge MUST be updated accordingly
+
+### Contact Requests
+
+**general**
+* A contact request it MUST appear in the "All" and "Contact Requests" sections.
+
+A contact request item MUST BE EITHER one initiated by the user OR one received by user.
+
+**contact requests initiated by user:**
+* If the user sends a contact request and it has not be accepted or rejected yet, then it MUST appear with the state "Sent"
+* If the user sends a contact request and it has been accepted, then it MUST appear with the state "Accepted"
+* If the user sends a contact request and it has been declined, then it MUST appear with the state "Declined"
+
+**contact requests received by user:**
+* The user MAY accept or reject a contact request in the activity center
+* The user MUST be able to accept a contact request directly in this type of activity item
+* The user MUST be able to decline & block a contact request directly in this type of activity item
+* Clicking the context menu of this type of activity item MUST show 3 options: "view profile" and "rename" and "Decline and Block"
+* Clicking "View profile" MUST open the profile modal for that user
+* Clicking "Rename" MUST open the profile modal for that user in that rename user section
+* Clicking "Decline and Block" MUST Reject the contact request
+* Clicking "Decline and Block" MUST Block that contact
+* If a contact request item is rejected it MUST display "Declined"
+* If a contact request item is rejected and blocked it MUST display "Declined & Blocked"
+* If a contact request item is accepted it MUST display "Accepted"
+* Clicking on this type of activity item MUST NOT do anything
+
+### Mentions
+
+* If the user is mentioned the message containing the message MUST appear in the "All" and "Mentions" sections.
+* The user MUST be able to mark this type of item as read
+* Clicking the context menu of this type of activity item MUST show 2 options: "view profile" and "Decline and Block"
```

Comment:
`Decline and block` is a weird name. You can't really decline a mention. Maybe it should be just named `Block user`?